client/src/App.js

```diff
@@ -14,6 +14,9 @@ import Templates from "./pages/Templates";
 import Prospects from "./pages/Prospects";
 import Campaigns from "./pages/Campaigns";
 import AddProspects from "./pages/AddProspects";
+import CharlieGoogle from "./pages/CharlieGoogle";
```

Comment:
What are these components? are they being used

Reply:
woops forgot to take that out. Resolved